packages/dids/src/did-dht.ts

```diff
@@ -284,7 +278,28 @@ export class DidDhtMethod implements DidMethod {
       };
     }
 
-    const didDocument = await DidDht.getDidDocument({ did: parsedDid.did });
+    let didDocument: DidDocument;
+
+    /**
+     * TODO: This is a temporary workaround for the following issue:  https://github.com/TBD54566975/web5-js/issues/331
+     * As of 5 Dec 2023, the `pkarr` library throws an error if the DID is not found. Until a
+     * better solution is found, catch the error and return a DID Resolution Result with an
+     * error message.
+     */
+    try {
+      didDocument = await DidDht.getDidDocument({ did: parsedDid.did });
+    } catch (error: any) {
+      return {
+        '@context'            : 'https://w3id.org/did-resolution/v1',
+        didDocument           : null,
+        didDocumentMetadata   : {},
+        didResolutionMetadata : {
+          contentType  : 'application/did+ld+json',
```

Comment:
it is not LD! just `application/did+json` should suffice

https://www.w3.org/TR/did-core/#application-did-json

Reply:
Done ✅